Cython/Compiler/ExprNodes.py

```diff
@@ -9850,10 +9857,18 @@ class GeneratorExpressionNode(LambdaNode):
     #
     # loop      ForStatNode   the for-loop, containing a YieldExprNode
     # def_node  DefNode       the underlying generator 'def' node
+    # call_parameters [ExprNode]   (Internal) parameters passed to the DefNode call
```

Comment:
Seeing this actually makes it seem weird that a `GeneratorExpressionNode` inherits from `LambdaNode`. It may not be outright wrong, but the first calls the function in order to create it. That's quite different. And it's what makes the difference here stick out.